Model

AleksandarSavic95 opened the pull request:
Finished model for TODO-item

AleksandarSavic95 committed:
#ALSA-23 setup jwt-auth + 2 users in database

- users added using a seeder

*note:* valid login request doesn't generate the jwt token

AleksandarSavic95 committed:
#ALSA-24 setup CORS

TODO: be strict with origin settings in cors.php

AleksandarSavic95 committed:
#ALSA-14 make TodoItem's model

- added Priority constants (low, mid, high),
- tested obtaining a constant value in web.php :)

calico-anne commented on database/migrations/2018_04_25_142709_create_todo_items_table.php:
```diff
@@ -0,0 +1,35 @@
+     */
+    public function up()
+    {
+        Schema::create('todo_items', function (Blueprint $table) {
```
Indentacija! :D

AleksandarSavic95 replied:
To se saamo tako indentovalo :D

calico-anne commented on database/migrations/2018_04_25_142709_create_todo_items_table.php:
```diff
@@ -0,0 +1,35 @@
+            $table->increments('id');
+            $table->timestamps();
+	        $table->string('title', 50);
+	        $table->string('content');
```
Mozes za `content` staviti `text`, 255 je max duzina stringa.

calico-anne commented on database/migrations/2018_04_25_142709_create_todo_items_table.php:
```diff
@@ -0,0 +1,35 @@
+	        $table->string('title', 50);
+	        $table->string('content');
+            $table->integer('priority');
+            $table->boolean('done');
```
Potrudi se za boolean-e da koristis nesto tipa `is_done`

AleksandarSavic95 committed:
#ALSA-14 fix todo_items table

- change `content` type to `text` (longer than `string`'s 255 chars)
- fixed indents from tabs to 4 spaces
- installed `doctrine/dbal` for DB column modification
